correlatoranalyser/fit.py

```diff
@@ -99,9 +104,49 @@ def calc_aug_chi2(self, nlf: lsqfit.nonlinear_fit) -> float:
 
     """save the interesting results from a lsqfit, if nbst is given then save in corresponding row nbst of the bootstrap parameters"""
 
+    def eval(self, abscissa: np.ndarray | None= None) -> dict:
+        r"""
+
+        """
+        
```

Comment:
Man könnte hier vlt. noch ein Check einbauen, dass self.fcn is not None, ansonsten der Hinweis, dasss man zuerst einen fit importiert. Was hälst du davon?